packages/compass-generative-ai/src/components/generative-ai-input.tsx

```diff
@@ -260,103 +329,110 @@ function GenerativeAIInput({
   return (
     <div className={containerStyles}>
       <div className={inputBarContainerStyles}>
-        <div className={inputContainerStyles}>
-          <TextInput
-            className={textInputStyles}
-            ref={promptTextInputRef}
-            sizeVariant="small"
-            data-testid="ai-user-text-input"
-            aria-label="Enter a plain text query that the AI will translate into MongoDB query language."
-            placeholder={placeholder}
-            value={aiPromptText}
-            onChange={(evt: React.ChangeEvent<HTMLInputElement>) =>
-              onChangeAIPromptText(evt.currentTarget.value)
-            }
-            onKeyDown={onTextInputKeyDown}
-          />
-          <div className={floatingButtonsContainerStyles}>
-            {isFetching ? (
-              <div className={loaderContainerStyles}>
-                <SpinLoader />
-              </div>
-            ) : showSuccess ? (
-              <div className={loaderContainerStyles}>
-                <Icon
-                  className={
-                    darkMode
-                      ? successIndicatorDarkModeStyles
-                      : successIndicatorLightModeStyles
-                  }
-                  glyph="CheckmarkWithCircle"
-                />
-              </div>
-            ) : (
-              <>
-                {aiPromptText && (
-                  <IconButton
-                    aria-label="Clear prompt"
-                    onClick={() => onChangeAIPromptText('')}
-                    data-testid="ai-text-clear-prompt"
-                  >
-                    <Icon glyph="X" />
-                  </IconButton>
-                )}
-              </>
-            )}
-            <Button
-              size="small"
-              className={cx(
-                generateButtonStyles,
-                !darkMode && generateButtonLightModeStyles
-              )}
-              disabled={!aiPromptText}
-              data-testid="ai-generate-button"
-              onClick={() =>
-                isFetching ? onCancelRequest() : handleSubmit(aiPromptText)
-              }
+        <div className={cx(inputContainerStyles)}>
+          <div className={cx(isFetching ? gradientAnimationStyles : null)}>
```

Comment:
Not sure why we are not showing this even when its not fetching. MongoDB docs shows the animation and it kinda looks nice.

But, when user generates and is focused on the input, the animation does not show due to (blue) focus ring. Can we disable it when its fetching so that user can see animation?

Reply:
Oh, thanks for sharing the slack link. I tried to play around, seems harder than i expected!